pkg/jsonlog/jsonlogbytes.go

```diff
@@ -23,8 +23,6 @@ func (mj *JSONLogBytes) MarshalJSONBuf(buf *bytes.Buffer) error {
 	if len(mj.Log) != 0 {
 		if first == true {
 			first = false
```

Comment:
since `first` is initialized to `true` here https://github.com/vdemeester/docker/blob/pkg-jsonlog-test-coverage/pkg/jsonlog/jsonlogbytes.go#L20 couldn't this become `first = false` directly? (maybe I'm missing something)